### PR TITLE
Double Precision Issue

### PR DESCRIPTION
**Pull request by Madrich**

### CODE REVIEW

File: code/AssetLib/3DS/3DSLoader.cpp [hunk 1]
```diff
@@ -449,7 +449,7 @@ void Discreet3DSImporter::ParseChunk(const char *name, unsigned int num) {
         // Read the lense angle
         camera->mHorizontalFOV = AI_DEG_TO_RAD(stream->GetF4());
         if (camera->mHorizontalFOV < 0.001f) {
-            camera->mHorizontalFOV = AI_DEG_TO_RAD(45.f);
+            camera->mHorizontalFOV = float(AI_DEG_TO_RAD(45.f));
```
Comment (kimkulling) [inline code/AssetLib/3DS/3DSLoader.cpp:452]: Please use ai_real instead

File: code/PostProcessing/CalcTangentsProcess.cpp [hunk 2]
```diff
@@ -56,7 +56,7 @@ using namespace Assimp;
 // ------------------------------------------------------------------------------------------------
 // Constructor to be privately used by Importer
 CalcTangentsProcess::CalcTangentsProcess() :
-        configMaxAngle(AI_DEG_TO_RAD(45.f)), configSourceUV(0) {
+        configMaxAngle(float(AI_DEG_TO_RAD(45.f))), configSourceUV(0) {
```
Comment (kimkulling) [inline code/PostProcessing/CalcTangentsProcess.cpp:59]: Please use ai_real instead

**Review by kimkulling — CHANGES_REQUESTED** [review]

File: code/AssetLib/3DS/3DSLoader.cpp (hunk 1 above)
Comment (Madrich) [reply code/AssetLib/3DS/3DSLoader.cpp:452]: mHorizontalFOV and configMaxAngle are floats by definition and should not be changed to double. The AI_DEG_TO_RAD return ai_real, so we need to convert to float.

File: code/PostProcessing/CalcTangentsProcess.cpp (hunk 2 above)
Comment (Madrich) [reply code/PostProcessing/CalcTangentsProcess.cpp:59]: same

Comment (kimkulling) [reply code/PostProcessing/CalcTangentsProcess.cpp:59]: Ok, I am fine with that.

File: code/AssetLib/3DS/3DSLoader.cpp (hunk 1 above)
Comment (kimkulling) [reply code/AssetLib/3DS/3DSLoader.cpp:452]: I am fine with that.

**Review by kimkulling — APPROVED** [review]

**Comment by kimkulling** [comment]
Merged, thanks a lot for your contribution.

**Pull request merged**